.travis.yml

```diff
@@ -1,11 +1,14 @@
+sudo: required
+dist: trusty
```

Comment:
@netmilk See my comment here: https://github.com/apiaryio/dredd/pull/310#discussion_r47811905 it's basically the new image on Travis CI that has modern compilers. The default is much older.